indico/htdocs/sass/modules/abstracts/_abstract_list.scss

```diff
@@ -24,3 +24,7 @@
         color: #777;
     }
 }
+
+#button-back {
```

Comment:
With class it gets overwritten by something else, but that doesn't matter much now since we still don't know where to put them.